src/index.ts

```diff
@@ -127,7 +180,8 @@ function getEmit(
     // Emit Javascript
     const output = instance.languageService.getEmitOutput(filePath);
 
-    loader.clearDependencies();
+    // Remove original dependency on resourcePath and replace with rawFilePath
```

Comment:
OK cool that makes sense.  So given we weren't preserving `dependencies` / `contextDependencies` in our original code it wasn't an issue because we would only be dealing with a single file in that context.  But this is not true in the world where `css-ts-loader` is chained before `ts-loader`.  Is that correct?